Code refactor

nickcat1 opened the pull request:
I decided to rewrite the script from scratch. Let me know if there are any issues.

nickcat1 committed:
Update list.txt

nickcat1 committed:
Update list.txt

RiRiSharp commented:
You forgot to add in the `ytd-reel-shelf-renderer` which shows shorts in the search results

ricardofauch commented:
even with ytd-reel-shelf-renderer its still not blocking shorts on search page.
i think the issue is the selector Title which is superior to ":has(h2:has-text(/^Shorts$/i))"

nickcat1 committed:
Update list.txt

This should work with youtube's update.
- Blocks "grid-shelf-view-model" (Shorts reel in search results)
- Blocks the 2nd "yt-chip-cloud-chip-renderer" if the text "Shorts" appears. (Shorts search button)

nickcat1 commented:
Thanks for letting me know. @ricardofauch
